src/components/Templates.vue

```diff
@@ -1,35 +1,33 @@
 <template>
   <div data-test="templates">
     <div class="templates-bar">
-      <TopBar :tabs="tabs">
-        <template v-slot:tool>
-          <button class="tool-btn" @click="importTemplate">
-            <font-awesome-icon class="icon" icon="file-import" />
-            Import template
-          </button>
-        </template>
-      </TopBar>
+      <p class="title">
+        Data Request Templates <span class="templates-number">{{ templates.length }}</span>
+      </p>
+      <el-button class="import-btn" @click="importTemplate" type="primary">
+        <font-awesome-icon class="icon" icon="file-import" />
+        Import template
+      </el-button>
     </div>
     <div class="centered">
       <div v-if="Object.entries(paginatedItems)" class="container-templates">
-        <div v-show="currentPage === 1" class="add">
-          <router-link to="/request/editor">
-            <img
-              data-test="create-template"
-              @click="createTemplate"
-              class="add-btn"
-              src="@/resources/svg/add.svg"
-            />
-          </router-link>
+        <div
+          data-test="create-template"
+          @click="createTemplateAndRedirect()"
+          v-show="currentPage === 1"
+          class="add"
+        >
+          <img class="add-btn" src="@/resources/svg/add.svg" />
+          <p class="text">Create a new template</p>
         </div>
         <TemplateCard
           v-for="template in paginatedItems"
           class="card"
           :name="template.name"
           :id="template.id"
+          :sources="template.radRequest.retrieve.length"
```

Comment:
Could you also add `:deployedTimes`?

Reply:
@aesedepece said this is not needed now. is it?

Reply:
Not yet IMO